docs/sources/api/_index.md

```diff
@@ -1094,3 +1093,21 @@ GET /prometheus/api/v1/alerts
 Prometheus-compatible rules endpoint to list all active alerts.
 
 For more information, please check out the Prometheus [alerts](https://prometheus.io/docs/prometheus/latest/querying/api/#alerts) documentation.
+
+## Compactor
+
+### `GET /compactor/ring`
+
+Displays a web page with the compactor hash ring status, including the state, healthy and last heartbeat time of each compactor.
+
+### `POST /loki/api/v1/delete`
+
+Create a new delete request. More details can be found in the [logs deletion documentation](../operations/storage/logs-deletion.md#request-log-entry-deletion).
+
+### `GET /loki/api/v1/delete`
+
+List all existing delete requests. More details can be found in the [logs deletion documentation](../operations/storage/logs-deletion.md#list-delete-requests).
```

Comment:
The handler for this only lists the deletes for the userID passed in the `X-Scope-OrgID` header. Is that something we should call out in the docs?

Reply:
Yes, that makes sense. The current text could be read as returning every delete request. It's been updated.